src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -209,8 +209,8 @@ private void AddLineBreaks(SourceText text, IList<string> lines)
         }
 
         private bool InsertOnEnterTyped(
-            SyntaxTree syntaxTree,
```

Comment:
:question: Why not pass `originalDocument` as the first argument, but keeping `syntaxTree` and `text` as they are now? It seems that would allow the continued use of `CompleteComment`.